test/Spaces/sphere.jl

```diff
@@ -66,18 +67,40 @@ end
             FT(3.97e-6),
             FT(6.77e-7),
         )
-        # Reference rtols with bubble w/ FT = Float64 (delete comment when fixed)
-        # bubble_rtols = (
-        #     FT(1.8),
-        #     FT(0.38),
-        #     FT(2.4e-5),
-        #     FT(0.0097),
-        #     FT(1.98e-8),
-        #     FT(0.00028),
-        #     FT(1.7e-11),
-        #     FT(7.94e-6),
-        #     FT(1.55e-14),
-        # )
+        no_bubble_rtols_f32 = (
+            FT(0.64f0),
+            FT(0.19f0),
+            FT(0.027f0),
+            FT(0.0049f0),
+            FT(0.0008f0),
+            FT(0.00014f0),
+            FT(2.4f-5),
+            FT(3.65f-6),
+            FT(4.05f-7),
+        )
+        # Reference rtols with bubble
+        bubble_rtols_f32 = (
+            FT(0.027),
+            FT(0.0008),
+            FT(2.4f-5),
+            FT(4.05f-7),
+            FT(1.35f-6),
+            FT(6.75f-8),
+            FT(1.08f-6),
+            FT(4.05f-7),
+            FT(6.75f-8),
+        )
+        bubble_rtols_f64 = (
+            FT(0.027),
+            FT(0.0008),
+            FT(2.33e-5),
+            FT(6.77e-7),
+            FT(1.98e-8),
+            FT(5.78e-10),
+            FT(1.7e-11),
+            FT(4.94e-13),
+            FT(1.53e-14),
+        )
```

Comment:
As I said on the call, we should increase the internal `high_order_Nq` so that these are all `O(eps(FT))`